src/shared/api/urls.ts

```diff
@@ -311,8 +311,15 @@ export function getG2SApiUrl() {
 export function getDigitalSlideArchiveMetaUrl(patientId: string) {
     return AppConfig.serverConfig.digital_slide_archive_meta_url + patientId;
 }
-export function getDigitalSlideArchiveIFrameUrl(patientId: string) {
-    return AppConfig.serverConfig.digital_slide_archive_iframe_url + patientId;
+export function getDigitalSlideArchiveIFrameUrl(
+    patientId: string,
+    studyId: string
+) {
+    const cancertype = studyId.split('_')[0];
+    return (
+        AppConfig.serverConfig.digital_slide_archive_iframe_url +
+        `#!/CDSA/${cancertype}/${patientId}`
+    );
 }
```

Comment:
this only works for TCGA studies, kinda forget where we are checking that but i tested a non TCGA study and it correctly hides that case.

There might be some for which there is a mismatch, I found this one:

https://deploy-preview-3779--cbioportalfrontend.netlify.app/patient/tissueImage?studyId=coadread_tcga_pan_can_atlas_2018&caseId=TCGA-3L-AA1B

coadread -> coad